src/widgets/dialogs/SettingsDialog.cpp

```diff
@@ -164,6 +165,7 @@ void SettingsDialog::addTabs()
     this->addTab([]{return new GeneralPage;},          "General",        ":/settings/about.svg");
     this->ui_.tabContainer->addSpacing(16);
     this->addTab([]{return new AccountsPage;},         "Accounts",       ":/settings/accounts.svg", SettingsTabId::Accounts);
+    this->addTab([]{return new AliasesPage;},         "Aliases",       ":/settings/accounts.svg");
```

Comment:
Rest of these are aligned
```suggestion
    this->addTab([]{return new AliasesPage;},          "Aliases",       ":/settings/accounts.svg");
```